<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-09T11:25:38 IST" name="Suite" finished-at="2022-09-09T11:26:05 IST" duration-ms="26208">
    <groups>
    </groups>
    <test started-at="2022-09-09T11:25:38 IST" name="Test" finished-at="2022-09-09T11:26:05 IST" duration-ms="26208">
      <class name="tests.Login">
        <test-method signature="testLogin(java.util.HashMap)[pri:0, instance:tests.Login@5a5a729f]" started-at="2022-09-09T11:25:39 IST" name="testLogin" data-provider="dataSupplier" finished-at="2022-09-09T11:25:53 IST" duration-ms="13976" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, ExpectedResult=Success, Username=amotooricapbat612@gmail.com, Browser=Chrome, Password=abcdefgh}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:976)
at org.testng.Assert.assertEquals(Assert.java:1010)
at tests.Login.testLogin(Login.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.Login@5a5a729f]" started-at="2022-09-09T11:25:53 IST" name="tearDown" finished-at="2022-09-09T11:25:53 IST" duration-ms="195" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="testLogin(java.util.HashMap)[pri:0, instance:tests.Login@5a5a729f]" started-at="2022-09-09T11:25:53 IST" name="testLogin" data-provider="dataSupplier" finished-at="2022-09-09T11:26:01 IST" duration-ms="7341" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, ExpectedResult=Success, Username=amotooricapbat62@gmail.com, Browser=Firefox, Password=abcde}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.Login@5a5a729f]" started-at="2022-09-09T11:26:01 IST" name="tearDown" finished-at="2022-09-09T11:26:02 IST" duration-ms="985" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="testLogin(java.util.HashMap)[pri:0, instance:tests.Login@5a5a729f]" started-at="2022-09-09T11:26:02 IST" name="testLogin" data-provider="dataSupplier" finished-at="2022-09-09T11:26:04 IST" duration-ms="2637" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, ExpectedResult=Success, Username=amotooricapbat63@gmail.com, Browser=Edge, Password=abcde}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.Login@5a5a729f]" started-at="2022-09-09T11:26:04 IST" name="tearDown" finished-at="2022-09-09T11:26:05 IST" duration-ms="330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.Login -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
